src/QueryResponseCommandFacility.cc

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file QueryResponseCommandFacility class Implementation
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have received with this code.
+ */
+
+#include "app-framework/QueryResponseCommandFacility.hh"
+
+#include <iostream>
+
+namespace appframework {
+
+int QueryResponseCommandFacility::listen(DAQProcess* process) {
+    try {
+        bool keepGoing = true;
+        while (keepGoing) {
+            std::cout << "Enter a command" << std::endl;
+            std::string comm;
+            std::getline(std::cin, comm);
+
+            if (comm == "quit") {
+                keepGoing = false;
```

Comment:
I'm fully aware that this is a place holder until we have a CCM interface. Yet there is a logical behaviour which I believe is dangerous. Having the interface to know the concept of the command "quit" so that it can let the process die is a bit dangerous: requires re implementation of the quit command in different places. 
I suggest this interface to be agnostic in terms of the command. The Command facility (and later the CCM) can simply ask the state to the DAQProcess and exit this loop once the state is something like "waiting to be killed". 
If we go toward this direction, we can give a sense to the concept of state at the level of the application framework and reduce the implementation of the CCM interface.